src/main/webapp/find-events.html

```diff
@@ -23,9 +24,37 @@
           <i class="fas fa-bars"></i>
       </div>
     </header>
-    <h1>find events page to be added</h1>
-    <div>add post</div>
-    <div>map</div>
-    <script src="./assets/script.js"></script>
+
+    <h1>find events</h1>
+
+    <div id="page-container">
+      <!-- Placeholder for map. -->
+      <div id="map-placeholder"></div>
+
+      <!-- Sample posts -->
+      <div id="all-posts">
+        <button id="create-post-button" type="button">create a post</button>
+        <div class="post-card">
+          <h2 class="card-title">Maker Club @ Benson Memorial Center</h2>
+          <h3 class="card-subtitle">Thai Food | 5:30-6:30pm</h3>
+          <p class="card-description">
+            Our annual all-hands meeting is ending at 5:30, and all leftover food will be open to the public for an hour afterwards. 
+            Please come by and take as much as you’d like :) 
+            We ask only that you wait until the meeting is completely over before entering the space. Thank you!
+          </p>
+        </div>
+        <div class="post-card">
+          <h2 class="card-title">ACM @ Kenna Hall</h2>
+          <h3 class="card-subtitle">Burritos | 6:00pm-8:00pm</h3>
+          <p class="card-description">
+            Come join the ACM for free burritos and to learn more about what our club does! All are welcome to join the club happenings, regardless of major or year. 
+            The event itself goes from 6:00pm - 7:00pm, and leftovers will be available until 8:00pm. We have vegatarian and halal options available.
+          </p>
+        </div>
+      </div>
+    <div>
+
+    <script> addPosts(); </script>
```

Comment:
What is the logic behind adding this call in the HTML vs into the onLoad function in the JS? It might make it a little harder to reason about the order that things are happening in if JS functions are being called from script tags

Reply:
+1

Reply:
Good point! I was just testing it separately but now that it works I've added it to onLoad()!